tests/BohrungControllerTest.cs

```diff
@@ -60,7 +77,92 @@ public async Task GetByIdAsync()
     public async Task GetByIdWithInexistentStandort()
     {
         var inexistentBohrungId = 61325584;
-        var actionResult = await controller.GetByIdAsync(inexistentBohrungId).ConfigureAwait(false);
+        var actionResult = await bohrungController.GetByIdAsync(inexistentBohrungId).ConfigureAwait(false);
         Assert.AreEqual(typeof(NotFoundResult), actionResult.Result.GetType());
     }
+
+    [TestMethod]
+    public async Task CreateEditAndDeleteAsyncWithoutGeometry()
+    {
+        // Add
+        var bohrung = new Bohrung
+        {
+            Bezeichnung = "PERFECTBOOK",
+            StandortId = standort.Id,
+            HAblenkung = 9,
+            HQualitaet = 3,
+        };
+
+        Assert.IsInstanceOfType(await bohrungController.CreateAsync(bohrung), typeof(CreatedAtActionResult));
+        bohrung = await context.Bohrungen.FindAsync(bohrung.Id);
+        Assert.AreEqual("PERFECTBOOK", bohrung.Bezeichnung);
+
+        // Edit
+        bohrung.Bezeichnung = "RESERVEOLIVE";
+        Assert.IsInstanceOfType(await bohrungController.EditAsync(bohrung), typeof(OkResult));
+        bohrung = await context.Bohrungen.FindAsync(bohrung.Id);
+        Assert.AreEqual("RESERVEOLIVE", bohrung.Bezeichnung);
+
+        // Delete
+        await bohrungController.DeleteAsync(bohrung.Id);
+        Assert.IsInstanceOfType((await bohrungController.GetByIdAsync(bohrung.Id)).Result, typeof(NotFoundResult));
+    }
+
+    [TestMethod]
+    public async Task CreateAndEditAsyncWithGeometryShouldUpdateStandortInformation()
```

Comment:
PP: Wenn nicht zuviel Aufwand in diesem Test mehrere Bohrungen hinzufügen, damit auch das Verhalten (v.a. beim Löschen) mit mehreren Grundbuchnummern abgedeckt ist.